core/node/storage/pg_stream_store.go

```diff
@@ -239,10 +232,10 @@ func (s *PostgresStreamStore) maintainSchemaLock(
 					LogError(dlog.FromCtx(ctx))
 				s.exitSignal <- err
 			}
-
-			if err = SleepWithContext(ctx, 1*time.Second); err != nil {
-				return
-			}
+		}
+		// Wait 1s between db pings.
```

Comment:
Comment is incorrect? But anyway it's redundant: it's pretty obvious from the code that this is sleep between pings.

Reply:
I'll take the comment out. In the meantime this PR is in a weird state because I'm trying to solicit more info about how the race condition bug is occurring and so it's regressing away from it's final state until I fix the issue causing the test flake.